boot/startup64.S

```diff
@@ -152,6 +152,25 @@ startup:
 
 	leaq	startup_stack_top(%rip), %rsp
 
+	# Disable x2APIC if supported & enabled on APs
+
+	movl	$0x1, %eax
+	cpuid
+	notl	%ecx
+	andl	$0x200000, %ecx
+	jnz	nox2apic				# x2APIC supported?
+	movq	$0x1b, %rcx
+	rdmsr
+	notl	%eax
+	andl	$0xc00, %eax		# x2APIC enabled?
```

Comment:
Here and below, word operations like I did as part of my NX changes ( #241 ) ?
andw $0xc00,%ax
andw $0xf3ff,%ax
orw $0x800,%ax